src/util/performance.js

```diff
@@ -172,15 +172,15 @@ export const PerformanceUtils = {
         return metrics;
     },
 
-    getWorkerPerformanceMetrics() {
+    getWorkerPerformanceMetrics(): { timeOrigin: string, measures: Array<{name: string, startMark?: string, endMark?: string}> } {
```

Comment:
I seems the right type for `performance.getEntriesByType(...)` should be this one:

```suggestion
    getWorkerPerformanceMetrics(): { timeOrigin: string, measures: Array<PerformanceEntry> } {
```
(It wouldn't error in any case because we're coercing from `any`)

Reply:
I was thinking that after the getEntriesbyName - thanks!